cppcache/src/PoolFactory.cpp

```diff
@@ -74,7 +74,7 @@ PoolFactory::PoolFactory(const Cache& cache)
 PoolFactory& PoolFactory::setFreeConnectionTimeout(
     std::chrono::milliseconds connectionTimeout) {
   if (connectionTimeout <= std::chrono::milliseconds::zero()) {
-    throw IllegalArgumentException("connectionTimeout must greater than 0.");
+    throw IllegalArgumentException("connectionTimeout must be greater than 0.");
```

Comment:
These strings are repetitive, how about we use a couple of named constants?  e.g.:

```
const std::string greaterThanZeroWarning = " must be greater than 0.";
...

throw IllegalArgumentException("connectionTimeout" + greaterThanZeroWarning);
```
Thanks for fixing the typos and grammar!

Reply:
I'd rather not use a constant and leave this as it it.
I think the code is more clear with the text, even if it is repetitive. Also if I use the constant, I would have to leave out the final stop in the sentence because at least one log add something after "greater than zero".